server/service/individualServices/LtpStructureUtility.js

```diff
@@ -9,12 +9,70 @@ const onfAttributes = require('onf-core-model-ap/applicationPattern/onfModel/con
  * @return {List} ltp list that matches the given layerProtocolName.
  */
 exports.getLtpsOfLayerProtocolNameFromLtpStructure = async function (LayerProtocolName, ltpStructure) {
-  let ltpsOfLayerProtocolName = [];
-  let ltpList = ltpStructure["core-model-1-4:control-construct"][0][onfAttributes.CONTROL_CONSTRUCT.LOGICAL_TERMINATION_POINT];
-  if (ltpList != undefined) {
-    ltpsOfLayerProtocolName = ltpList.filter(ltp =>
-      ltp[onfAttributes.LOGICAL_TERMINATION_POINT.LAYER_PROTOCOL].find(layerProtocol =>
-        layerProtocol[onfAttributes.LAYER_PROTOCOL.LAYER_PROTOCOL_NAME] === LayerProtocolName))
-  }
-  return ltpsOfLayerProtocolName;
+    let ltpsOfLayerProtocolName = [];
+    let ltpList = ltpStructure["core-model-1-4:control-construct"][0][onfAttributes.CONTROL_CONSTRUCT.LOGICAL_TERMINATION_POINT];
+    if (ltpList != undefined) {
+        ltpsOfLayerProtocolName = ltpList.filter(ltp =>
+            ltp[onfAttributes.LOGICAL_TERMINATION_POINT.LAYER_PROTOCOL].find(layerProtocol =>
+                layerProtocol[onfAttributes.LAYER_PROTOCOL.LAYER_PROTOCOL_NAME] === LayerProtocolName))
+    }
+    return ltpsOfLayerProtocolName;
+}
+
+/**
+ * This function fetches LTP instance from LtpStructure if uuid is equal to given.
+ * @param {String} uuid identifier of the logical-termination-point instance.
+ * @param {Object} ltpStructure control construct.
+ * @return {Object} ltp that matches the given uuid.
+ */
+async function getLtpForUuidFromLtpStructure(uuid, ltpStructure) {
+    let ltp = {};
+    let ltpList = ltpStructure["core-model-1-4:control-construct"][onfAttributes.CONTROL_CONSTRUCT.LOGICAL_TERMINATION_POINT];
```

Comment:
As controlConstruct is an array can we change it to ["core-model-1-4:control-cosntruct"][0]

Reply:
Thanks for pointing it out @vanithavalluripalli9 . Updated CC to array.